portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +24,24 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    /*response.setContentType("text/html;");
+    response.getWriter().println("<h1>Hello Oyin!</h1>");*/
+
+    ArrayList<String> comments = new ArrayList<String>();
```

Comment:
General Java style: If you don't explicitly need comments to be ArrayList (e.g. need to pass it to a function that requires ArrayList), we usually just declare it as the more generic version, List

Also, depending on your version of java, you shouldn't have to declare <String> again in the constructor.

`List<String> comments = new ArrayList<>();`

The compiler assumes you want `new ArrayList<String>()` because you declared it to be a String List already